core/src/autogluon/core/utils/utils.py

```diff
@@ -378,6 +382,9 @@ def infer_eval_metric(problem_type: str) -> Scorer:
         return accuracy
     elif problem_type == MULTICLASS:
         return accuracy
+    elif problem_type == QUANTILE:
+        from autogluon.core.metrics.quantile_metrics import pinball_loss
```

Comment:
why not import at top of file like the other metrics?

Reply:
Yes, I moved it to the top. 
```
from ..metrics import accuracy, root_mean_squared_error, Scorer
from ..metrics.quantile_metrics import pinball_loss
```